assets/js/sections/contesting.js

```diff
@@ -208,26 +213,43 @@ $('#start_date').change(function () {
 
 // On Key up check and suggest callsigns
 $("#callsign").keyup(function () {
-	var call = $(this).val();
+	let call = $(this).val().toUpperCase();
 	if (call.length >= 3) {
 
-		$.ajax({
-			url: 'lookup/scp',
-			method: 'POST',
-			data: {
-				callsign: $(this).val().toUpperCase()
-			},
-			success: function (result) {
-				$('.callsign-suggestions').text(result);
-				highlight(call.toUpperCase());
-			}
-		});
+        if ( scp_data.request == "" || ! call.startsWith(scp_data.request) ) {
```

Comment:
This should be `===`

Reply:
As `scp_data.request` always is a string, `==` is okay here.

Reply:
no it's not, it needs to `===`.  Please read up on javascript best practices before submitting PR's for changes to javascript.